.github/ISSUE_TEMPLATE/api_references_issue.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 📗 Having issues with Sequelize API reference?
+about: Let us know about any unclear, missing, or problematic Sequelize API reference.
+title: ""
+labels: "type: typescript"
+assignees: ""
+---
+
+<!--
+If you don't follow the issue template, your issue may be closed.
+Please note this is an issue tracker, not a support forum.
+For general questions, please use StackOverflow.
+-->
+
+## Issue Creation Checklist
+
+[ ] I have read the [contribution guidelines](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)
```

Comment:
I don't think these contribution guidelines provide any useful information to users that want to report documentation issues